src/app.js

```diff
@@ -1 +1,56 @@
 'use strict';
+
+const fs = require('fs');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let pathFile = '';
+let contentFile = '';
+
+const question1 = () => {
+  return new Promise((resolve) => {
+    rl.question('Please enter the file name ', (answer) => {
+      if (answer.length === 0) {
+        console.log('Error');
+        rl.close();
+
+        return;
+      };
```

Comment:
```suggestion
      }
```
Unnecessary semicolon